www/src/components/diagram.js

```diff
@@ -316,8 +323,30 @@ const Diagram = () => (
       >
         <ItemTitle>Static Web Host</ItemTitle>
         <ItemDescription>
-          Amazon S3, Netlify, GitHub Pages, Surge.sh, Aerobatic, Now.sh, & many
-          more
+          <ItemDescriptionLink to="/docs/deploying-to-s3-cloudfront">
```

Comment:
Since this is structured data now, maybe we can make it a yaml file or something else that we can query against? e.g. consider one of the following approaches!


*docs/hosts.yml*

```yaml
- title: Amazon S3
  url: /docs/deploying-to-s3-cloudfront
- title: Netlify
  url: /docs/hosting-on-netlify
# etc
```